docs/resources/User.md

```diff
@@ -28,7 +28,8 @@ There are other rules and restrictions not shared here for the sake of spam and
 | ------------- | --------- | ---------------------------------------------------------------------------------------------------- | --------------------- |
 | id            | snowflake | the user's id                                                                                        | identify              |
 | username      | string    | the user's username, not unique across the platform                                                  | identify              |
-| discriminator | string    | the user's 4-digit discord-tag                                                                       | identify              |
+| discriminator | string    | the user's Discord-tag                                                                               | identify              |
+| global_name   | ?string   | the user's display name, if it is set. For bots, this is the application name                        | identify              |
```

Comment:
```suggestion
| global_name?  | ?string   | the user's display name, if it is set. For bots, this is the application name                        | identify              |
```
this seems to be missing in the `author` field of the message returned when [executing a webhook](https://discord.com/developers/docs/resources/webhook#execute-webhook)